routes/message.js

```diff
@@ -7,10 +7,11 @@ const isEntityValid = require('~/middlewares/entityValidation')
 const messageController = require('~/controllers/message')
 const Chat = require('~/models/chat')
 
-const body = [{ model: Chat, idName: 'chat' }]
+const params = [{ model: Chat, idName: 'id' }]
 
 router.use(authMiddleware)
 
-router.post('/', isEntityValid({ body }), asyncWrapper(messageController.sendMessage))
+router.get('/', isEntityValid({ params }), asyncWrapper(messageController.getMessages))
+router.post('/', isEntityValid({ params }), asyncWrapper(messageController.sendMessage))
```

Comment:
Why did you change entity validation from body to params?

Reply:
cause now id in params, not in body)